ui/admin/app/templates/scopes/scope/host-catalogs/index.hbs

```diff
@@ -36,46 +36,44 @@
 
     {{#if @model}}
 
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each @model as |hostCatalog|}}
-            <body.row as |row|>
-              <row.headerCell>
-                {{#if (can 'read host-catalog' hostCatalog)}}
-                  <LinkTo
-                    @route='scopes.scope.host-catalogs.host-catalog'
-                    @model={{hostCatalog.id}}
-                  >
-                    {{hostCatalog.displayName}}
-                  </LinkTo>
-                {{else}}
-                  {{hostCatalog.displayName}}
-                {{/if}}
-                <p>{{hostCatalog.description}}</p>
-              </row.headerCell>
-              <row.cell>
-                <HostCatalogTypeBadge @model={{hostCatalog}} />
-              </row.cell>
-              <row.cell>
-                <Copyable
-                  @text={{hostCatalog.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
+      <Hds::Table
+        @model={{@model}}
+        @columns={{array
+          (hash label=(t 'form.name.label'))
+          (hash label=(t 'form.type.label'))
+          (hash label=(t 'form.id.label'))
+        }}
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              {{#if (can 'read host-catalog' B.data)}}
+                <LinkTo
+                  @route='scopes.scope.host-catalogs.host-catalog'
+                  @model={{B.data.id}}
                 >
-                  <code>{{hostCatalog.id}}</code>
-                </Copyable>
-              </row.cell>
-            </body.row>
-          {{/each}}
-        </table.body>
-      </Rose::Table>
+                  {{B.data.displayName}}
+                </LinkTo>
+              {{else}}
+                {{B.data.displayName}}
+              {{/if}}
+              <p>{{B.data.description}}</p>
+            </B.Td>
+            <B.Td>
+              <HostCatalogTypeBadge @model={{B.data}} />
+            </B.Td>
+            <B.Td>
+              <Copyable
```

Comment:
Can we use [HDS::Copy::Button](https://helios.hashicorp.design/components/copy/button?tab=code) here?

Reply:
@calcaide I had these changes committed yesterday failed to push them up.🙈.  This one will be `Hds::Copy::Snippet`